x-pack/test/api_integration/apis/monitoring/_health/fixtures/response_es_beats.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+
+export const esBeatsResponse = (date = moment().format('YYYY.MM.DD')) => {
```

Comment:
What's the reason for using today's date rather than a fixed date as the default?

Maybe to not have to change the date in the UI?

Reply:
In integration tests we start with a pristine state, so when we insert these documents in the (empty) data stream it will create a backing indice to store the data and that backing indice will have the current day's date as convention
see https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html#data-streams-generation